README.md

```diff
@@ -169,12 +173,21 @@ import mdformat
 
 unformatted = "content...\n"
 # Pass in `extensions` here! It is an iterable of extensions that should be loaded
-formatted = mdformat.text(unformatted, extensions={"table"})
+formatted = mdformat.text(unformatted, extensions={"tables"})
 ```
 
 Read the [contribution guide](<https://github.com/executablebooks/mdformat/blob/master/CONTRIBUTING.md#developing-code-formatter-plugins>)
 if you wish to implement a new parser extension plugin.
 
 ### Existing formatter plugins
 
-Coming soon!
+<table>
+  <tr>
+    <th>Plugin</th>
+    <th>Syntax Extensions</th>
+  </tr>
+  <tr>
+    <td><a href="https://github.com/executablebooks/mdformat-tables">mdformat-tables</a></td>
+    <td><code>table</code></td>
```

Comment:
Should this be `tables` not `table`? I think we want to have the string here that's defined in https://github.com/executablebooks/mdformat-tables/blob/8bd4129826ab000cde8ab756045e99d17cf5c607/pyproject.toml#L30